DefaultTabBar.js

```diff
@@ -71,7 +71,7 @@ var DefaultTabBar = React.createClass({
     });
 
     return (
-      <View style={[styles.tabs, {backgroundColor : this.props.backgroundColor || null}]}>
+      <View style={[this.props.style, styles.tabs, {backgroundColor : this.props.backgroundColor || null}]}>
```

Comment:
revers order preferable imho. We give ability to overwrite any property, it is useful.